com.unity.render-pipelines.core/Editor/CoreEditorUtils.cs

```diff
@@ -128,6 +128,14 @@ public static void DrawFixMeBox(string text, MessageType messageType, Action act
         public static void DrawMultipleFields(string label, SerializedProperty[] ppts, GUIContent[] lbls)
             => DrawMultipleFields(EditorGUIUtility.TrTextContent(label), ppts, lbls);
 
+        private static float GetLongestLabelWidth(GUIContent[] lbls)
```

Comment:
could we rename this variable from `lbls `to `labels`? i know other functions have this terminology but i think we can progressively improve readability